src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,69 @@
+package pages;
+
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+
+public class LoginPage {
+
+    public LoginPage(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(how = How.ID, using = "identifierId")
+    public WebElement userNameField;
+
+    @FindBy(how = How.CLASS_NAME, using = "CwaK9")
+    public WebElement nextButton;
+
+    @FindBy(how = How.NAME, using = "password")
+    public WebElement passwordField;
+
+    @FindBy(how = How.CLASS_NAME, using = "goog-imageless-button-content")
+    public WebElement calendarButton;
+
+    @FindBy(how = How.XPATH, using = "//SPAN[@class='gb_bb gbii']")
+    public WebElement picturOfMe;
+
+    @FindBy(how = How.XPATH, using = "//A[@id='gb_71']")
+    public WebElement signOut;
+
+    public void enterEmailAddres(String username) {
+        userNameField.sendKeys(username);
+    }
+
+    public void enterPassword(String password) {
+        passwordField.sendKeys(password);
+    }
+
+    public void clickNext() {
+        nextButton.click();
+    }
+
+    public void passwordFieldIsDispayed() {
+        Assert.assertEquals(passwordField.isDisplayed(), true);
+    }
+
+    public void calendarButtonIsDisplayed() {
```

Comment:
Just a minor suggestion. You could have a return type in these methods. If they return their classes e.g.: LoginPage in this case and just use "return this;" at the end of the method. You have a fluent api of the page object. So when you call these methods you can use it like: 
loginPage.enterEmailAddres("youraddress").enterPassword("your password").clickNext();
much easier to read in the stepdefs or inter layers.